src/Caching/StackExchangeRedis/test/Infrastructure/RedisXunitTestExecutor.cs

```diff
@@ -1,59 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
The buidl complains about these files since it's not commented out correctly. Deleting it seems preferable to correcting it since the files have been untouched for about 4 years now: https://github.com/dotnet/aspnetcore/commit/4a743980f7e6abca6e149bdff2e2002e420480b1#diff-70562989a3df743d9bd95761dfb5210a3a92b0783c3af443ec80a4469f2ab155R9